src/archivist/services/pageDeclaration.test.js

```diff
@@ -158,4 +158,66 @@ describe('PageDeclaration', () => {
       });
     });
   });
+
+  describe('toJSON', () => {
+    it('convert noise and content semectors to JSON', async () => {
```

Comment:
```suggestion
    it('converts noise and content selectors to JSON', async () => {
```

Reply:
By the way, I think to JSON is misnamed because it not only converts the representation to JSON but also renames its attributes to be the same as those found in service declarations.

Reply:
I check if it works well in different page declaration configurations
- select and remove
- filter and select

Reply:
I do not see the added value of splitting this. Why not `select` and `executeClientScript`? Grouping `select` with `remove` and `select` with `filter` seems arbitrary to me, I do not see the domain constraint on this. 

I'm in favor of a single test that checks that all attributes are properly converted, or else make a test for each attribute, but not an arbitrary grouping.

Reply:
I thus made two tests
- one with simple declaration (fetch and select) to make sure other fields are undefined
- one with all fields